doc/oemof_solph.rst

```diff
@@ -432,11 +432,67 @@ Furthermore, an efficiency for loading, unloading and a capacity loss per time i
         capacity_loss=0.001, nominal_capacity=50,
         inflow_conversion_factor=0.98, outflow_conversion_factor=0.8)
 
+
+Using the investment object with the GenericStorage component
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Based on the `GenericStorage` object the `GenericInvestmentStorageBlock` adds two main investment possibilities.
+
+    *	Invest into the flow parameters e.g. a turbine or a pump
+    *	Invest into capacity of the storage  e.g. a basin or a battery cell
+    
+As an addition to other flow-investments, the storage class implements the possibility to couple or decouple the flows 
+with the capacity of the storage. 
+Three parameters are responsible for connecting the flows and the capacity of the storage:
+
+    *	' `invest_relation_input_capacity` ' fixes the input flow investment to the capacity investment. A ratio of ‘1’ means that the storage can be emptied within one time-period. 
+    *	' `invest_relation_output_capacity` ' fixes the output flow investment to the capacity investment. A ratio of ‘1’ means that the storage can be emptied within one period.
+    *	' `invest_relation_input_output` ' fixes the input flow investment to the output flow investment. For values <1, the input will be smaller and for values >1 the input flow will be larger. 
+    
+You should not set all 3 parameters at the same time, since it will lead to overditermination
+This following example-storage is not usable. The input flow has to be 1/6 of the capacity whereas the output has to be equal to the capacity and additionally both flows shall be equal.
+
+.. code-block:: python
+
+    solph.GenericStorage(
+        label='storage',
+        inputs={b_el: solph.Flow(investment= solph.Investment(ep_costs=100))},
+        outputs={b_el: solph.Flow(investment= solph.Investment(ep_costs=100)},
+        capacity_loss=0.001, 
+        inflow_conversion_factor=0.98, outflow_conversion_factor=0.8),
+        invest_relation_input_output = 1
+        invest_relation_input_output = 1, invest_relation_input_capacity = 1/6,
```

Comment:
There seems to be a typo. One `invest_relation_input_output` should be `invest_relation_output_capacity`.